tests/conftest.py

```diff
@@ -202,3 +202,61 @@ def persisted_task(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="module")
+def ai_voltage_chan_with_excit(any_x_series_device):
```

Comment:
FYI, if we don't expect to reuse a fixture in other tests, we don't have to define it in conftest.py. We can define it in test_channel_properties.py.